Java/Endpoint Examples/Multipart Payload/CompressedPdf.java

```diff
@@ -10,7 +10,7 @@
 import okhttp3.Response;
 import org.json.JSONObject;
 
-public class CompressedPdf {
+public class CompressedPDF {
```

Comment:
Changing the capitalization here breaks the sample -- the class name must match the name of the `.java` file it's declared in.